packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -80,29 +85,15 @@ export function _publishCloudEventsSend(
   events: CloudEvent[],
   options: PublishCloudEventsOptionalParams = { requestOptions: {} },
 ): StreamableMethod {
-  return context.path("/topics/{topicName}:publish", topicName).post({
-    ...operationOptionsToRequestParameters(options),
-    contentType:
-      (options.contentType as any) ??
-      "application/cloudevents-batch+json; charset=utf-8",
-    body: (events ?? []).map((p) => {
-      return {
-        id: p["id"],
-        source: p["source"],
-        data: p["data"],
-        data_base64:
-          p["dataBase64"] !== undefined
-            ? uint8ArrayToString(p["dataBase64"], "base64")
-            : undefined,
-        type: p["type"],
-        time: p["time"]?.toISOString(),
-        specversion: p["specversion"],
-        dataschema: p["dataschema"],
-        datacontenttype: p["datacontenttype"],
-        subject: p["subject"],
-      };
-    }),
-  });
+  return context
+    .path("/topics/{topicName}:publish", topicName)
+    .post({
+      ...operationOptionsToRequestParameters(options),
+      contentType:
+        (options.contentType as any) ??
+        "application/cloudevents-batch+json; charset=utf-8",
+      body: events,
```

Comment:
this change doesn't look correct. we should definitely have serializer for this type as it has uint8array type properties.